src/test/java/GmailResponseHelperImplTest.java

```diff
@@ -12,19 +12,150 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import com.google.api.services.gmail.model.Message;
+import com.google.api.services.gmail.model.MessagePart;
+import com.google.api.services.gmail.model.MessagePartHeader;
+import com.google.common.collect.ImmutableList;
 import com.google.sps.model.GmailResponseHelper;
 import com.google.sps.model.GmailResponseHelperImpl;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
 import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 import org.junit.Assert;
 import org.junit.Test;
 
 /**
  * Tests the GmailResponseHelperImpl and ensures that statistics for the GmailResponse are
  * calculated correctly
  */
-public class GmailResponseHelperImplTest implements GmailTestBase {
+public class GmailResponseHelperImplTest {
   private static final GmailResponseHelper gmailResponseHelper = new GmailResponseHelperImpl();
 
+  int DEFAULT_N_DAYS = 7;
```

Comment:
You forgot to prefix the constants with `static final`

Reply:
Thanks! Fixed